services/movie.py

```diff
@@ -0,0 +1,40 @@
+from typing import List
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: List[int] = None,
+        actors_ids: List[int] = None

```

Comment:
If the variable can be None you must show this fact in type-hint with the help `Optional[your variable type]` or 'your variable type | None'. 
For example: `genres_ids: Optional[List[int]] = None` or `genres_ids: List[int] | None = None`